tutorials/.ipynb_checkpoints/forest_usage-checkpoint.ipynb

```diff
@@ -0,0 +1,579 @@
+{
```

Comment:
Do you actually need to include this file? Shouldn't `forest_usage.ipynb` be sufficient?

Reply:
Yeah, the ipynb document should be sufficient. Thanks for catching that!